### ISSUE TITLE
Signup page - allow for some custom text/announcements

### ISSUE THREAD

**Issue by Antik79** [LemmyNet/lemmy-ui#1990]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

We would like to be able to add some information that would be easy to change to the signup page.  We want to include some information about the federation and provide a list with other instances to sign up to.   So it would be helpful to be able to do this from within Lemmy without having to revert our changes after every update.

**Comment by brihuang99** [comment]
![Screenshot_20230801_152959](https://github.com/LemmyNet/lemmy-ui/assets/122472324/bf770151-6f9d-4277-a3bc-60ae62defa1d)

I see that lemmy.ml has this alert banner on their sign up page; the other instances I've looked at do not have that. Not sure how this alert banner is currently implemented, but it would be nice to allow admins to turn this on and customize it as they see fit.

**Comment by Nutomic** [comment]
One option is to include this in the registration question.

**Comment by dessalines** [comment]
Yep, the registration question is the correct place to put this.
